src/legacy/ui/public/capabilities/react/index.ts

```diff
@@ -19,4 +19,4 @@
 
 export { UICapabilitiesProvider } from './ui_capabilities_provider';
 export { injectUICapabilities } from './inject_ui_capabilities';
-export { UICapabilities } from '../ui_capabilities';
+export { Capabilities } from '../../../../../core/public/capabilities';
```

Comment:
Was the change to export `Capabilities` from the NP intentional here? Previously, I was exporting `UICapabilities` from here, so instead of doing:

```
import { UICapabilities } from 'ui/capabilities';
import { injectUICapabilities } from 'ui/capabilities/react';
```

you could just do the following, as a convenience:
```
import { injectUICapabilities, UICapabilities } from 'ui/capabilities/react';
```

There are a few places where we had to change the usages because of this change.

Reply:
Ah, my intent was to remove the export altogether. It felt more correct to import `UICapabilities` from `ui/capabilities`, since it's not a react-specific entity. I'm not opposed to reverting though.

Reply:
Yeah... I went back and forth on this a few times, and I'm not tied to the way it was being handled previously. It just seemed like we shouldn't be exporting the NP `Capabilities`.

Reply:
I agree. I'll remove the stray export and leave the rest as-is then